cmd/gapit/dump_pipeline.go

```diff
@@ -79,9 +77,9 @@ func (verb *pipeVerb) getBoundPipelineResource(ctx context.Context, c client.Cli
 		return nil, err
 	}
 
-	targetType := api.Pipeline_GRAPHICS
+	targetType := "Graphics Pipeline"
```

Comment:
Why are we moving back to string literals?

Reply:
Andrew believes it will make it more extensible.

Reply:
I don't think this is going to need extending anytime soon. I can understand strings for the stages, but the pipeline type should stay an enum.